examples/tenzir-developers/docker-compose.yaml

```
@@ -96,6 +99,11 @@ services:
       - '5432:5432'
     volumes:
       - postgres_data:/var/lib/postgresql/data
+    healthcheck:
+      test: [ 'CMD-SHELL', 'pg_isready -U postgres' ]
```

We use the long-form for arrays everywhere else in our YAML files. Mind rewriting all three instances of this?
```suggestion
      test:
        - CMD-SHELL
        - pg_isready -U postgres
```

---

:man_shrugging: